service/lib/agama/autoyast/converter.rb

```diff
@@ -59,6 +60,7 @@ class Converter
         "root",
         "scripts",
         "security",
+        "services-manager",
```

Comment:
Sorry, but you are not supposed to add the `services-manager`to this list. These are the Agama sections, not the AutoYaST ones. You need to extend the `ScriptsReader` instead.